### DISCUSSION TITLE
Why I can't use mouse?

### DISCUSSION THREAD

**Discussion by winkee01** [neovide/neovide#1295] [Q&A]
When I use neovide to open a file, if `:set mouse=a` is not set, the mouse can't be used. I just want to use the mouse to select and copy text (like I do in the iTerm2 terminal), but not be able to. I can do `:set mouse=a`, but since it's related to vim, I don't want to use the mouse to interact with vim, but only with the UI, simply put, I use the mouse only when I want to select some text and do the copy to my system clipboard.

**Comment by MultisampledNight** [comment]
See #1204. For the time being, there is no such option, but you can craft a replacement for the clipboard part with Vimscript.

**Reply by winkee01** [reply]
Sorry, what I want is that mouse selection does not interact with vim, but instead with the UI itself, as I do in iTerm2 terminal, I can select anything that appears in the iTerm2 terminal.
![image](https://user-images.githubusercontent.com/49930785/166396238-4d0a7655-b661-4c48-a257-7714dd38a347.png)

Why I don't want the selection limited to vim? because in vim I can only yank things in the edit area, I can't yank things that appear in the floating window, status line, and other places that are not in the edit area, this is so limited. So what I want is that anything appear in the UI, I can use my mouse to do the selection, just like in iTerm2.
